mailpoet/assets/js/src/automation/templates/components/template-detail.tsx

```diff
@@ -74,6 +74,14 @@ export function TemplateDetail({
             <Button icon={chevronRight} />
           </div>
           <div className="mailpoet-automation-template-detail-footer-actions">
+            {error && (
```

Comment:
Should we add a way for the user to dismiss the error message or make it disappear after some time?